internal/flags/test_client.go

```diff
@@ -168,3 +168,14 @@ func (_ *FakeClient) StringValueDetails(_ context.Context, _ string, _ string,
 	_ openfeature.EvaluationContext, _ ...openfeature.Option) (openfeature.StringEvaluationDetails, error) {
 	panic("unimplemented")
 }
+
+
+// State implements openfeature.IClient.
+func (_ *FakeClient) State() openfeature.State {
+	panic("unimplemented")
+}
+
+// Track implements openfeature.IClient.
+func (_ *FakeClient) Track(_ context.Context, _ string, _ openfeature.EvaluationContext, _ openfeature.TrackingEventDetails) {
+	panic("unimplemented")
+}
```

Comment:
Any way we could mock this instead?

Reply:
Actually, it occurred to me that we could define a narrower slice of the interface (we basically need `IClient.Boolean`), but I'd prefer to do that as a separate refactor.  I could do that before this merge if you'd prefer.

Reply:
Let's do that

Reply:
#5493 for that